addons/l10n_sa_edi/views/account_journal_views.xml

```diff
@@ -0,0 +1,115 @@
+            <field name="arch" type="xml">
+                <xpath expr="//notebook" position="inside">
+                    <field name="l10n_sa_csr" invisible="1"/>
+                    <field name="l10n_sa_compliance_csid_json" invisible="1"/>
+                    <field name="l10n_sa_production_csid_json" invisible="1"/>
+                    <field name="l10n_sa_compliance_checks_passed" invisible="1"/>
+                    <page name="zatca_einvoicing" string="ZATCA" attrs="{'invisible': [('country_code', '!=', 'SA')]}">
+                        <group>
+                            <group>
+                                <field name="l10n_sa_serial_number"/>
+                            </group>
+                        </group>
+                        <p>
+                            <b>
+                                In order to be able to submit Invoices to ZATCA, the following steps need to be completed:
+                            </b>
+                            <ol class="mt-2 mb-4">
+                                <li>
+                                    Set a Serial Number for your device
+                                    <i class="fa fa-check text-success ml-1"
+                                       attrs="{'invisible': [('l10n_sa_serial_number', '=', False)]}"/>
+                                </li>
+                                <li>
+                                    Request a Compliance Certificate (CCSID)
+                                    <i class="fa fa-check text-success ml-1"
+                                       attrs="{'invisible': [('l10n_sa_compliance_csid_json', '=', False)]}"/>
+                                </li>
+                                <li>
+                                    Complete the Compliance Checks
+                                    <i class="fa fa-check text-success ml-1"
+                                       attrs="{'invisible': [('l10n_sa_compliance_checks_passed', '=', False)]}"/>
+                                </li>
+                                <li>
+                                    Request a Production Certificate (PCSID)
+                                    <i class="fa fa-check text-success ml-1"
+                                       attrs="{'invisible': [('l10n_sa_production_csid_json', '=', False)]}"/>
+                                </li>
+                            </ol>
+                        </p>
+                        <div class="alert alert-info d-flex justify-content-between align-items-center" role="alert"
+                             attrs="{'invisible':['|', ('l10n_sa_csr', '!=', False), ('l10n_sa_serial_number', '=', False)]}">
+                            <p class="mb-0">
+                                Generate a CSR before starting the onboarding process.
+                            </p>
+                            <button name="l10n_sa_regen_csr" type="object" icon="fa-key"
```

Comment:
@jco-odoo There is an option on the fatoora portal the the onboarding again for a solution that has already been onboarded. In this case, it is useful to have a new CSR since each time we onboard, we need to do it with a new CSR